channel_acceptor.go

```diff
@@ -139,5 +140,38 @@ func (s *ChannelAcceptor) acceptChannel(_ context.Context,
 		}, nil
 	}
 
+	switch expectedChanBid.ChannelType {
+	// The bid doesn't have specific requirements for the channel type.
+	case order.ChannelTypePeerDependent:
+
```

Comment:
Style nit: add a `break` here to make it more explicit that we're not falling through (one of the quirk's w/ Go's `switch` statements IMO).